brainiak/reconstruct/__init__.py

```diff
@@ -0,0 +1,14 @@
+#  Copyright 2018 David Huberdeau & Peter Kok
```

Comment:
and Vy Vo?

Reply:
I am just going to attribute authorship to myself in the main script. I don't need to claim copyright. Thank you for raising the issue.